tests/interpreters/test_riscv_emulator.py

```diff
@@ -10,7 +10,7 @@
 
 pytest.importorskip("riscemu", reason="riscemu is an optional dependency")
 
-from xdsl.interpreters.riscv_emulator import RV_Debug, run_riscv
+from xdsl.interpreters.riscv_emulator import RV_Debug, run_riscv  # noqa: E402
```

Comment:
what rule is this?

Reply:
[Import module](https://beta.ruff.rs/docs/rules/module-import-not-at-top-of-file/)

According to [PEP 8](https://peps.python.org/pep-0008/#imports), "imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants."

Reply:
Can you please move it to the top in a separate PE or ignore the rule in the project?

Reply:
Ah ok this makes sense. @kingiler can you please add a comment with this explanation? Then I think it's good to merge

Reply:
I'm not sure what comment to add? There are quite some `# noqa:` in the code base already without comments.